test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -76,6 +80,72 @@ public class RequestTests
             }
         }
 
+        [Fact]
+        public async Task LargeMultipartUpload()
```

Comment:
I could add a new `ITestCondition` to `Microsoft.AspNetCore.Testing`. Something like `BitnessSkipCondition`. But is that really necessary? Skipping on Mono shouldn't be so bad since we're still running the test on CoreCLR.